lib/cbus_elixir/accounts/accounts.ex

```diff
@@ -0,0 +1,81 @@
+defmodule CbusElixir.Accounts do
+  @moduledoc """
+  The boundary for the Accounts system.
+  """
+
+  import Ecto.{Query, Changeset}, warn: false
+  alias CbusElixir.{Accounts.User, Repo}
+
+  def list_users do
+    Repo.all(User)
+  end
+
+  def list_users_paged(params) do
+    User
+      |> Repo.paginate(params)
+  end
+
+  def get(id), do: Repo.get(User, id)
+
+  def get_by(%{"email" => email}) do
+    Repo.get_by(User, email: email)
+  end
+
+  def create_user(attrs) do
+    %User{}
+    |> User.create_changeset(attrs)
+    |> Repo.insert()
+  end
+
+  def update_user(%User{} = user, attrs) do
+    user
+    |> User.changeset(attrs)
+    |> Repo.update()
+  end
+
+  def update_user_by_id(id, attrs) do
+    get(id)
+    |> User.changeset(attrs)
+    |> Repo.update()
+  end
+
+  def delete_user(%User{} = user) do
+    Repo.delete(user)
+  end
+
+  def change_user(%User{} = user) do
+    User.changeset(user, %{})
+  end
+
+  def list_sessions(user_id) do
+    with user when is_map(user) <- Repo.get(User, user_id), do: user.sessions
```

Comment:
I'm not sure what this will return if there's no `else` to match this `with`... I feel like the error should be handled or `[]` should be returned.